types/FontConnectorTypes.ts

```diff
@@ -8,8 +8,8 @@ export enum FontDownloadType {
 export type Font = {
     id: string;
     name: string;
-    fontFamily?: string | null;
-    fontStyle?: string | null;
+    family?: string | null;
+    style?: string | null;
```

Comment:
can it be undefined or is it always null?

Reply:
@pietervp

Reply:
I think `null` would be the only supported value. This type is part of the connector contract which demands this property as a return type.

Reply:
then the ? can be removed